ci/scripts/fetch_changelog.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+// Fetches content of a first opened GitHub issue labeled "Release Changelog"
+
+const fetch = require("node-fetch");
+
+fetch("https://api.github.com/repos/trezor/trezor-suite/issues?labels=Release%20Changelog")
+.then(response => response.json())
+.then(data => {
+  console.log(data[0]["body"])
+})
+.catch(error => console.log("Changelog issue not found"))
```

Comment:
``` 
const fetch = require('node-fetch');

fetch('https://api.github.com/repos/trezor/trezor-suite/issues?labels=Release%20Changelog')
    .then(response => response.json())
    .then(data => {
        console.log(data[0].body);
    })
    .catch(() => console.log('Changelog issue not found'));
```
Code is fine, just updated it to our style.

Reply:
4277febbf0db7a43ee678eaaadbbb5f17f038038